README.md

```diff
@@ -854,6 +855,120 @@ registration: {
+      baseUrl: 'https://acme.okta.com',
+      clientId: '{{myClientId}}', // REQUIRED
+      registration: {
+        parseSchema: function(schema, onSuccess, onFailure) {
+           // handle parseSchema callback
+        },
+        preSubmit: function (postData, onSuccess, onFailure) {
+           // handle preSubmit callback
+        },
+        postSubmit: function (response, onSuccess, onFailure) {
+            // handle postsubmit callback
+        }
+      },
+      features: {
+        // Used to enable registration feature on the widget. https://github.com/okta/okta-signin-widget#feature-flags
+         registration: true // REQUIRED
+      }
+    });
+```
+
+Optional configuration:
+
+- **parseSchema:** Callback used to mold the JSON schema that comes back from Okta API.
+
+    ```javascript
+    // The callback function is passed 3 arguments: schema, onSuccess, onFailure
+    // 1) schema: json schema returned from the API.
+    // 2) onSuccess: success callback.
+    // 3) onFailure: failure callback. Note: accepts an errorObject that can be used to show form level or field level errors.
+
+    parseSchema: function (schema, onSuccess, onFailure) {
+      // This example will add an additional field to the registration form
+        schema.profileSchema.properties.address = {
+          'type': 'string',
+          'description': 'Street Address',
+          'default': 'Enter your street address',
+          'maxLength': 255
+        };
+        schema.profileSchema.fieldOrder.push('address');
+        onSuccess(schema);
+    }
+    ```
+ - **preSubmit:** Callback used primarily to modify the request parameters sent to Okta API.
+
+    ```javascript
+     // The callback function is passed 3 arguments: postData, onSuccess, onFailure
+     // 1) postData: form data that will be posted to the registration API.
+     // 2) onSuccess: success callback.
+     // 3) onFailure: failure callback. Note: accepts a errorObject that can be used to show form level or field level errors.
+    preSubmit: function (postData, onSuccess, onFailure) {
+      // This example will add @companyname.com to the email if user fails to add it during registration
+      if (postData.username.indexOf('@acme.com') > 1) {
+        return postData.username;
+      } else {
+        return postData.username + '@acme.com';
+      }
+    }
+    ```
+ - **postSubmit:** Callback used to primarily get control and to modify the behavior post submission to registration API .
+
+    ```javascript
+     // The callback function is passed 3 arguments: response, onSuccess, onFailure
+     // 1) response: response returned from the API post registration.
+     // 2) onSuccess: success callback.
+     // 3) onFailure: failure callback. Note: accepts an errorObject that can be used to show form level or field level errors.
+    postSubmit: function (response, onSuccess, onFailure) {
+      // This example will log the server response to the browser console
```

Comment:
Since we are logging and returning the response, lets make sure we update the comment indicating what exactly is happening.

Reply:
updated the comment